app/javascript/src/components/Dashboard/Notes/utils.js

```diff
@@ -0,0 +1,14 @@
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+
+import { NOTES_NAVLINKS } from "./constants";
+
+dayjs.extend(relativeTime);
```

Comment:
@navaneethsdk Pay attention to the docs. https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=How%20to%20extend%20day.js%3F

If you don't agree with the docs then lemme know. If you agree with the docs, then you should be making the change as mentioned in the docs.

Reply:
I am sorry for overlooking this. Perhaps it did not come to my mind while extending the library. I will ensure I won't repeat this mistake. I completely agree with it.